metagraph/tests/algorithms/__init__.py

```diff
@@ -138,7 +149,9 @@ def _compare_values(self, expected_val, ret_val, algo_path, rel_tol, abs_tol):
         )
         if issubclass(expected_type, ConcreteType):
             try:
-                compare_val = self.resolver.translate(ret_val, type(expected_val))
+                compare_val = self._translate_atomic_type(
+                    ret_val, type(expected_val), algo_path
+                )
```

Comment:
Using `_translate_atomic_type` here removes the need for the `except` block in line 170.

Reply:
https://github.com/metagraph-dev/metagraph/pull/117/commits/8351d986d41c2cafc9c57d25e645cd0b88a956df removes the unnecessary try/except block.